domainhunter.py

```diff
@@ -609,12 +622,17 @@ def getIndex(cells, index):
             # Perform domain reputation checks
             if check:
                 
+		unwantedResults = ['Uncategorized','error','Not found.','Spam','Spam URLs','Pornography','badurl','Suspicious','Malicious Sources/Malnets','captcha','Phishing','Placeholders']
+		
                 bluecoat = checkBluecoat(domain)
-                print("[+] {}: {}".format(domain, bluecoat))
+		if bluecoat not in unwantedResults:
+                    print("[+] Bluecoat - {}: {}".format(domain, bluecoat))
                 ibmxforce = checkIBMXForce(domain)
-                print("[+] {}: {}".format(domain, ibmxforce))
+		if ibmxforce not in unwantedResults:
+                    print("[+] IBM XForce - {}: {}".format(domain, ibmxforce))
```

Comment:
Not Python3 syntax (change to use spaces vs tabs)
```./domainhunter.py -s expireddomains.net
  File "./domainhunter.py", line 628
    if bluecoat not in unwantedResults:```

Reply:
tabs fixed